docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+# mLRS Documentation: Quick Start Guide #
+
+([back to main page](../README.md))
+
+This page provides hints, suggestions and links to help get new users started with mLRS quickly.
+
+This guide covers the use case where mLRS is used to provide a single radio link to carry both RC control and full bi-directional MAVLink Telemetry.  When we discuss configuration of your flight controller, we assume you are running ArduPilot.  We also assume you use EdgeTX or OpenTX and that you will connect a ground station and/or use the [Yaapu Telemetry App](CRSF.md).
+
+We expect that you are starting fresh and that all ArduPilot and mLRS parameters not covered here still have their default settings.
+
+The steps here should be followed in order and when practical we provide a way to confirm a step was completed successfully.
+
+Some notation and terminology:
+- We use "Tx module" and "receiver" to refer to a mLRS module in the Tx/transmitter role and Rx/receiver role. In the use case assumed here, the Tx module will be installed in a EdgeTX or OpenTX radio handset, and the receiver will be mounted on the vehicle and connected to the flight controller.
+- We use lower case "rx" and "tx" when referring to the receive and transmit pads or pins of a serial port connection.
+
+## 1. Choose and purchase a Tx module and a compatible receiver
+
+There are many supported choices of mLRS hardware, including off-the-shelf options and  many DIY designs.
+We cover here three popular but mutually incompatible commercially available off-the-shelf options.  We summarize some pros and cons below to help you choose. Don't mix Tx module and receiver between these options.
+
+### MatekSys mLRS [mR900-30-TX](https://www.mateksys.com/?portfolio=mr900-30-tx) module kit with their [mR900-30](https://www.mateksys.com/?portfolio=mr900-30) receiver
+- These 900 MHz band modules were designed specifically for mLRS and arrive pre-flashed with mLRS.
+- This option should provide the longest range of the three discussed here.
+- The mR900-30-TX module kit includes an HC04 module for a convenient Bluetooth serial connection to your ground station software running on a computer or Android device.
+- These modules are especially easy to flash via DFU over their USB-C connection.
+- These use the SX126x LoRa chip and support 19 Hz, 31 Hz and FSK(50 Hz) modes.
+- This option does not work with any ELRS receivers.
+- The Tx module kit does not include an antenna or a JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- The [mR900-22](https://www.mateksys.com/?portfolio=mr900-22) is a compatible lower power receiver which might be appropriate for smaller vehicles.  But it is not currently recommended because it is not yet easy to flash.
```

Comment:
I guess I would keep a note to new users
 "... not currently recommended for new users because ..."

Reply:
If I'm going to mention it at all, I want to mention why they might want to choose it in spite if it not being recommended.

Reply:
I just think like so it sounds a bit rude. But your call really :)

Reply:
@jlpoltrack does it sound OK to you?
It doesn't read rude to me, but, of course, it wouldn't because I wrote it and didn't intend to be rude to anyone.  I want to understand where I went wrong.  
Did my first version also sound rude?  Rude to whom?
Am I being too hard on MatekSys, like they should have made it easier to flash?  This is not what I think, I like this receiver and currently have it in a build and I want new users to know it is available if they need light weight.  It's our problem to make it easier to flash.  I just don't think we should recommend it until we do.

Reply:
Except it can't be flashed via passthrough with current ardupilot releases and it would have to be unsoldered to use a USB TTL adapter (which may come as a surprise to new users) and now it's getting long again and closer to what I had in the first place which Olli objected to.

Reply:
Ah, good point.  My suggestions would then be:
- Second sentence could just be 'However, it doesn't have a USB-C port for flashing firmware.'
- Remove the bullet entirely - if you think it's not recommended then it doesn't necessarily need to be mentioned IMO

Reply:
I don't like the first suggestion because, while it is factually correct, not having a USB-C port should not be grounds to reject a receiver in my opinion.  None of the ELRS receivers have USB-C but they can be flashed via passthrough and I recommend some of them here.  I think that once ArduPilot releases my parity fix, the mR900-22 can be recommended.

Removing the bullet entirely is a good suggestion, but I really wanted to mention this receiver because users may find it when they look at the  rest of the MatekSys mLRS line.  I want to acknowledge it exists and how it is different from the other MatekSys 900 MHz receiver and say why we don't recommend it yet.  This is similar to why I mentioned the R9m receivers even though they are not recommended; because I consider them to be more difficult to flash than the ELRS receivers but I also expect new users might have them and want to know why I didn't recommend them.

BTW, it's really nice that MatekSys released a light weight 900MHz receiver because there are plenty of off the shelf compatible ELRS 2.4GHz receivers, but 900MHz ELRS receivers are not compatible with the MatekSys Tx modules.  This is another reason I wanted to mention this receiver here.  Because this receiver exists, the MatekSys 900MHz Tx module is more attractive for users like me who care about weight.  I don't want to new users to think the R9M is the only option for off-the-shelf light weight 900MHz.

 I think what I have now says exactly what I want to say. @jlpoltrack does it sound rude to you or is there some other problem with how I'm saying this?

Reply:
I can see the rude argument.  I try to suggest alternatives so how about including 'new users' like is done in the R9 section:

'But it is not currently recommended for new users because it is not yet easy to flash.'

Reply:
OK.  I looked up the word rude just to be sure I wasn't missing something.  I wonder if when you guys use the word rude here,  you mean the tone is too abrupt or sharp?  In addition to the adding 'new users' which does soften the tone a bit, I've added a bit more information so the reader can make an informed choice.  How is this?

- The [mR900-22](https://www.mateksys.com/?portfolio=mr900-22) is a compatible lower power receiver which is appropriate for smaller vehicles.  But it is not currently recommended for new users because it is not yet easy to flash without disconnecting it from your flight controller.

Reply:
Works for me

Reply:
I also changed "might be appropriate" to "is appropriate" which I guess also gives a more positive tone.

Reply:
in terms of ease a USB-C is currently by a long mile the most easiest method to flash ... also and especially thx to the web flasher ... I really can't see how this possibly could be disputed
for esp32 this might be different but as a matter of fact the matek 22 is a stm32

I suggest

"But it is not recommended for new users because flashing is curretly not as easy as for other options"

Reply:
Who is disputing that?  But once Ardupilot releases passthrough parity support, we can also make updating the 22 quite easy.  Just install python and run the [flash_rx_via_ap_passthru.py](https://github.com/olliw42/mLRS/blob/dev-flashscript/tools/flash_rx_via_ap_passthru.py) script.  Or better yet, add support for serial passthrough flashing to the web flasher.

What is wrong with the text I write above and which I already pushed?  What's wrong with hinting at what the current difficulty is so the reader can decide for themselves?  Maybe the user doesn't mind unsoldering or maybe their flight controller has a connector they can unplug to flash.

Reply:
well - and that's just my feeling reading it - the whole fact that it is mentioned and then just "turned" down with some explanation which I would think a new user won't really understand, is a bit odd ... after all it's a chapter on the tx kit and 30 receiver ... why is it mentioned at all 

I wonder if one shouldn't just remove the bullet

Reply:
> Just install python and run the [flash_rx_via_ap_passthru.py](https://github.com/olliw42/mLRS/blob/dev-flashscript/tools/flash_rx_via_ap_passthru.py) script. Or better yet, add support for serial passthrough flashing to the web flasher.

yo, everyone will want to first install python, and all the other dependencies needed for the script ... :)

btw, getting this into the web flasher will be a major effort ... I doubt it will happen by the time 4.6 becomes official

Reply:
with all that said I'd like to reiterate that I just share my 2 cents, you are free to ignore me :)

Reply:
I do very much appreciate your 2 cents.  It's worth much more than that!  :-)  But I'd like to leave this part as I have in my last push.

I also think we should merge the python flashing script.  The ELRS configurator apparently depends on python.  What are they doing to make it easy to install?

Reply:
as you know I'm very hesitant to publish it
I myself won't want to also maintain this, so I won't ... and I think it's not good to merge code where it is already clear today it will lack maintance

no idea what ELRS is doing

Reply:
when @jlpoltrack is happy, time for merge